test/schemas/validate/Schema1.Form.schema

```diff
@@ -0,0 +1,5 @@
+Form({
+  Args: {
```

Comment:
What is `Args` for forms?

Reply:
I think all fields should be passed to an action. Why need no `Fields` and `Args`

Reply:
There is no way to define arguments on an action, it was decided previously that arguments to an `Action`s should be defined on a corresponding `Form`.